packages/core/src/Tooltip/Tooltip.tsx

```diff
@@ -126,7 +126,8 @@ const Tooltip: React.FC<InferProps<typeof propTypes>> = ({ children, ...props })
   )
 }
 
-Tooltip.propTypes = propTypes
-Tooltip.defaultProps = defaultProps
+_Tooltip.defaultProps = defaultProps
+
+const Tooltip = withTheme(_Tooltip)
```

Comment:
n-b not sure withTheme is needed

Reply:
Left it there just in case. Worth looking at our usage of `withTheme` everywhere at some point